src/app/ngrx/reducers/episode-alltime-downloads.reducer.ts

```diff
@@ -4,12 +4,20 @@ import { AllActions, ActionTypes } from '../actions';
 
 export type State = EntityState<EpisodeAllTimeDownloads>;
 
-export const adapter: EntityAdapter<EpisodeAllTimeDownloads> = createEntityAdapter<EpisodeAllTimeDownloads>({
-  selectId: (e: EpisodeAllTimeDownloads) => e.guid,
```

Comment:
I had some changes to make to ngrx for the upgrade, so I also stopped using this selectId entityAdapter helper. I feel like it wasn't actually doing me any favors. The `id` property was still being added the the entity records under the hood. It would show up in marble tests and you'd have to account for it being there when it wasn't on the interface to the entity record. It just wasn't helping as much as I thought it would.